cypress/tests/integration/Categories.cy.js

```diff
@@ -6,7 +6,7 @@
  * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
  */
 
- describe('Tests categories in the submission wizard', function() {
+ describe.skip('Tests categories in the submission wizard', function() {
```

Comment:
Was the skip forgotten or is the test broken?

Reply:
All test related to categories are skipped, and will be updated on a separate issue